ros-kinetic/moveit_commander/moveit_commander-0.9.11.ebuild

```diff
@@ -22,7 +22,7 @@ RDEPEND="
 	ros-kinetic/shape_msgs
 	ros-kinetic/tf
 	dev-lang/python
-	dev-libs/assimp
+	media-libs/assimp
```

Comment:
I found the offending line in rosdistro.

https://github.com/ros/rosdistro/blob/d5f95b9efbef482e33d4b490d7d00f9a7f7465c7/rosdep/python.yaml#L2144

If it gets fixed there, then the fix will be permanent. I'll leave this to you unless you specify otherwise.

At that point, you can check this will be avoided permanently by doing the following.

```
rosdep update
superflore-gen-ebuilds --only moveit_commander --ros-distro kinetic
```

That will generate the ebuild, sign the manifest, and PR the changes for you.

Reply:
Opened pull request in rosdistro ros/rosdistro#17070 thanks!